server/game/cards/01-DTR/RailroadStation.js

```diff
@@ -0,0 +1,37 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class RailroadStation extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Railroad Station',
+            playType: ['noon'],
+            cost: ability.costs.bootSelf(),
+            target:{
+                activePromptTitle: 'Select dude to move from this location',
+                cardCondition: { location: 'play area', controller: 'current', condition: card => card.locationCard === this },
+                cardType: ['dude']
+            },
+            handler: context => {
+                this.game.promptForLocation(context.player, {
+                    activePromptTitle: 'Select destination',
+                    waitingPromptTitle: 'Waiting for opponent to select location',
+                    onSelect: (player, location) => {
+                        this.game.resolveGameAction(GameActions.moveDude({ 
+                            card: context.target, 
+                            targetUuid: location.uuid, 
+                            options: { needToBoot: false, allowBooted: true }
```

Comment:
Nice 😄 Your first card, but I hope you understand I cannot just let it without any comment 😛 The `options` can be removed as those are default. Just to make it simpler.

Reply:
after some struggle ... it's gone :)